funnel/templates/js/calendar_snippet.js.jinja2

```diff
@@ -0,0 +1,8 @@
+{% macro project_countdown() %}
+  {% raw %}
+    <span>{{ gettext('Today') }}</span>
+    <span>{{ gettext('Tomorrow') }}</span>
+    <span>{{ gettext('Day after') }}</span>
+    <span>{{ gettext('In %d days', 3) }}</span>
+  {% endraw %}
+{% endmacro %}
```

Comment:
If this template is not used anywhere, it may be accidentally removed in future. I've had a few PRs already this year that removed unused templates that I happened to notice.

Reply:
Translation-only templates can be kept in a special file named `translations_only.js.jinja2`, with a comment on each string pointing to where it is used.